src/main/java/org/terasology/navgraph/NavGraphSystem.java

```diff
@@ -79,6 +83,13 @@ public void update(float delta) {
         }
     }
 
+    /**
+     * Called whenever a Block is changed in the terrain
+     * @param pos position of the changed block
```

Comment:
Capitalization is inconsistent between the "Block" in the first line and "block" in the later ones